### PR TITLE
[GPU] Make static and const methods if possible

### PR DESCRIPTION
**Pull request by GermanAizek**
Nothing special, just an improvement.

### CODE REVIEW

File: GPU/Common/FramebufferManagerCommon.h [hunk 1]
```diff
@@ -512,15 +512,15 @@ class FramebufferManagerCommon {
...
+	static void NotifyRenderFramebufferUpdated(VirtualFramebuffer *vfb);
 	void NotifyRenderFramebufferSwitched(VirtualFramebuffer *prevVfb, VirtualFramebuffer *vfb, bool isClearingDepth);
 
 	void BlitFramebufferDepth(VirtualFramebuffer *src, VirtualFramebuffer *dst);
 
 	void ResizeFramebufFBO(VirtualFramebuffer *vfb, int w, int h, bool force = false, bool skipCopy = false);
 
-	bool ShouldDownloadFramebufferColor(const VirtualFramebuffer *vfb) const;
-	bool ShouldDownloadFramebufferDepth(const VirtualFramebuffer *vfb) const;
+	static bool ShouldDownloadFramebufferColor(const VirtualFramebuffer *vfb) ;
```
Comment (Nemoumbra) [inline GPU/Common/FramebufferManagerCommon.h:522]: Did you use clang-format?
`(const VirtualFramebuffer *vfb) ;`
`-------------------------------^`

**Comment by hrydgard** [comment]
I'm a big fan of making functions const or static when possible, it's important part of the principles of self-documenting code, and makes many refactorings easier. So this one is good. Just needs the little formatting fix above.

**Review by hrydgard — APPROVED** [review]

File: GPU/Common/FramebufferManagerCommon.h (hunk 1 above)
Comment (GermanAizek) [reply GPU/Common/FramebufferManagerCommon.h]: fix it https://github.com/hrydgard/ppsspp/pull/18986/commits/b57dab28126ec2ee5d9d62d9e6e37b5b756e667f

**Comment by GermanAizek** [comment]
> I'm a big fan of making functions const or static when possible, it's important part of the principles of self-documenting code, and makes many refactorings easier. So this one is good. Just needs the little formatting fix above.

fix it https://github.com/hrydgard/ppsspp/pull/18986/commits/b57dab28126ec2ee5d9d62d9e6e37b5b756e667f

**Pull request merged**